What's new in 8.8 (#3363)
elastic/security-docs: docs/whats-new.asciidoc

[float]
== Alerts enhancements 

*Control alert notifications and summaries*

The following enhancements give more control over how and when alert notifications are sent. For more information, refer to {security-guide}/rules-ui-create.html#rule-notifications[Set up alert notifications]. 

* You can now specify how often {security-guide}/rules-ui-create.html#rule-notifications[alert notifications] are sent to third-party systems (such as Slack, JIRA, email, etc.). You can apply your preferred frequency to all rule actions, or set notification frequency individually for each action. 
+
[role="screenshot"]
image::whats-new/images/8.8/action-frequency.png[Rule action frequency]
+
* You can decide whether to be notified each time an alert is generated, or receive alert summaries. 

* Instead of turning rules off to stop alert notifications, you can {security-guide}/rules-ui-management.html#snooze-rule-actions[snooze rule actions] for a specified time period. When you snooze rule actions, the rule continues to run on its defined schedule, but won’t perform any actions or send alert notifications. 

*Max alerts warning*

> When a rule reaches the maximum number of alerts it can generate in a single rule execution, the following {security-guide}/alerts-ui-monitor.html#troubleshoot-max-alerts[warning] is displayed on the rule’s details page and in the rule execution log: `This rule reached the maximum alert limit for the rule execution. Some alerts were not created.` To troubleshoot this event, we recommend you investigate alerts for any anomalies. For more information, refer to {security-guide}/alerts-ui-manage.html[Manage detection alerts].

Reviewer: ```suggestion
When a rule reaches the maximum number of alerts it can generate in a single rule execution, the following {security-guide}/alerts-ui-monitor.html#troubleshoot-max-alerts[warning] is displayed on the rule’s details page and in the rule execution log: `This rule reached the maximum alert limit for the rule execution. Some alerts were not created.` To troubleshoot this event, we recommend you check for unexpected alerts. For more information, refer to {security-guide}/alerts-ui-monitor.html#troubleshoot-max-alerts[Troubleshoot maximum alerts warning].
```

[float]
== Visualization actions and inline actions added to more places in the {security-app}

{security-guide}/es-ui-overview.html#visualization-actions[Visualization actions], which allow you to examine {es} queries used to retrieve data throughout the {security-app} or perform actions for the selected visualization, have been added to several places in the {security-app}. Look for the Inspect button (image:whats-new/images/8.8/inspect-icon.png[Inspect icon,16,16]) or options menu (image:whats-new/images/8.8/three-dot-menu.png[Options menu icon,17,17]) in the UI. 

{security-guide}/es-ui-overview.html#inline-actions[Inline actions] are displayed when you hover over a specific data field or value and allow you to customize your view or investigate further. They've also been added to more places throughout the {security-app}, such as:

* Explore pages (Host, Network, and User pages)
* Entity analytics (Entity Analytics Dashboard, user risk score, and host risk score features)
* Alerts and events table
* Event details flyout

[role="screenshot"]
image::whats-new/images/8.8/inline-actions-menu.png[Inline actions menu]

[float]
== Cloud Security enhancements

> *New Container Workload Protection (CWP) (beta)*

Reviewer 2: ```suggestion
*New Container Workload Protection (beta)*
```
Trying to avoid calling it CWP because it's ambiguous between Cloud workload and Container workload protection 😭

> You can now use {agent} to {security-guide}/d4c-overview.html[monitor and protect your containers] and prevent container drift. This solution uses a new integration, *Defend for Containers* (D4C), which allows you to create custom alerting and enforcement policies. 

Reviewer 2: ```suggestion
You can now use {agent} to {security-guide}/d4c-overview.html[protect your containers] by detecting and preventing malicious behavior and malware, and to capture workload telemetry data. This solution uses a new integration, *Defend for Containers* (D4C), which allows you to create custom alerting and enforcement policies. 
```

> *New Cloud Native Vulnerability Management(CNVM) (beta)*

Reviewer 2: ```suggestion
*New Cloud Native Vulnerability Management (CNVM) (beta)*
```

> The {security-guide}/vuln-management-overview.html[Cloud Native Vulnerability Management (CNVM)] feature helps you identify known vulnerabilities in your cloud workloads. When it finds vulnerabilities, it enables your remediation efforts by providing metadata such as the CVE identifier, CVSS score, severity, affected package, and a fix version if available, as well as information about impacted systems. 

Reviewer 2: ```suggestion
The {security-guide}/vuln-management-overview.html[Cloud Native Vulnerability Management (CNVM)] feature helps you identify known vulnerabilities in your cloud workloads. When it finds vulnerabilities, it enables your remediation efforts by providing metadata such as the CVSS, severity, affected package, and a fix version if available, as well as information about impacted systems. 
```
